### ISSUE TITLE
No Tutorial?

### ISSUE THREAD

**Issue by TerminalSS** [uzerjamal/Keylogger-detector#3]
There is no guide or setup process for this at all ?

**Comment by oplopr01** [comment]
me too looking for that,  did you get it!!

**Comment by TerminalSS** [comment]
> me too looking for that,  did you get it!!

Nope sorry i gave up on looking at this anyway

**Comment by uzerjamal** [comment]
Sorry about that!
This was built during a class over a couple of weeks during college and i never really came back to it until now..
I have updated documentation aswell as resolved a few other things
You likely are not looking for this now, but i wanna thank you for your interest many years ago!

**Comment by oplopr01** [comment]
Yeah u r correct..  Anyway thanks for the effort to write the reply mail🙂
